structuredheadings/plugin.js

```diff
@@ -369,8 +364,10 @@
             var block = elementPath.block;
 
             if (block && isNumbered(editor, block)) {
-              this.setValue(editor.config.autonumberCurrentStyle,
-                        editor.config.autonumberCurrentStyle);
+              this.setValue(
+                self.currentPreset,
+                self.currentPreset
```

Comment:
fwiw, I have no guess what this function call does. `setValue` seems very vabue.